scss/atoms/typography/_typography.scss

```diff
@@ -12,10 +12,10 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( max-width: $viewport-md-min ) {
 		font-size: 16px;
```

Comment:
Not mobile first currently, We can have that done in another PR. I simply used the variables instead of direct pixel values.